### ISSUE TITLE
[references] Add training script for binary classification

### ISSUE THREAD

**Issue by frgfm** [pyronear/pyro-vision#3]
## 🚀 Feature
Add training script for binary classification model

## Motivation

In order to have comparable evaluation runs of training, a training script can serve as a template for comparison and reproducibility.

## Pitch

The training script could be called with optional arguments that would tweak the different parameters according to the user's preferences.

**Comment by blenzi** [comment]
When running ```%run references/classification/torch/train.py --data-path data --epochs 0```

Locally (no GPU):
```
AssertionError: Torch not compiled with CUDA enabled
```
In colab (with GPU):
```
AttributeError                            Traceback (most recent call last)

/content/PyroNear/references/classification/torch/train.py in <module>()
    255     args = parser.parse_args()
    256 
--> 257     main(args)

/content/PyroNear/references/classification/torch/train.py in main(args)
    186 
    187     # Scheduler
--> 188     lr_scheduler = optim.lr_scheduler.OneCycleLR(optimizer, max_lr=args.lr,
    189                                               epochs=args.epochs, steps_per_epoch=len(train_loader),
    190                                               cycle_momentum=(not isinstance(optimizer, optim.Adam)),

AttributeError: module 'torch.optim.lr_scheduler' has no attribute 'OneCycleLR'
```

*Suggestion:* unify the default parameters on all scripts. E.g.: `--data-path`:  `./data` in fastai, `./tmp_data` in torch

**Comment by frgfm** [comment]
@blenzi Thanks for the feedback!
**Locally (no GPU)**
Add the following argument `--device cpu` (default is set to cuda)

**In colab (with GPU)**
I have to update the requirements of pytorch for this script (`OneCycleLR` is available since torch 1.3.0 and I guess colab hasn't updated its default version since the release). Try to force install pytorch 1.3.0 in colab for now ;)

Good catch for `--data-path`, I'm updating this right now!

**Comment by blenzi** [comment]
Ok, locally it seems to work after updating torch and:
```%run references/classification/torch/train.py --data-path data --epochs 1 --workers 0 --device cpu```

To avoid the problem on CPU, a default option `--device auto` could be used to detect the presence of a GPU or not.
E.g. for tensorflow: https://stackoverflow.com/questions/38559755/how-to-get-current-available-gpus-in-tensorflow

Could not find immediately an equivalent for torch but trying `torch.cuda` should raise `AttributeError`. I can open an issue later.
